src/PrizeVault.sol

```diff
@@ -496,26 +528,39 @@ contract PrizeVault is TwabERC20, Claimable, IERC4626, ILiquidationSource, Ownab
         }
     }
 
+    /**
+     * @notice Transfers yield fee shares to the yield fee recipient
+     * @dev Will revert if the caller is not the yield fee recipient or if zero shares are withdrawn
+     */
+    function claimYieldFeeShares(uint256 _shares) external onlyYieldFeeRecipient {
+        if (_shares == 0) revert MintZeroShares();
+
+        uint256 _yieldFeeBalance = yieldFeeBalance;
+        if (_shares > _yieldFeeBalance) revert SharesExceedsYieldFeeBalance(_shares, _yieldFeeBalance);
+
+        yieldFeeBalance -= _yieldFeeBalance;
+        _mint(yieldFeeRecipient, _shares);
```

Comment:
Should emit an event here.  Any major state change should include an event!  The underlying Transfer event isn't sufficient to determine the origin I believe

Reply:
Added `ClaimYieldFeeShares` event here: https://github.com/GenerationSoftware/pt-v5-vault/pull/75/commits/a2c73dada1ad2cf5276b1966722217479f83899c#diff-3f32973f6e271309aabc2174e0559d7ed0b537ec3e75f6c34022b3db7d083190R552